build_tools/cmake/ctest_all.sh

```diff
@@ -92,6 +92,14 @@ if [[ "$OSTYPE" =~ ^msys ]]; then
   )
 fi
 
+# TODO(laurenzo): figure out how to run samples with custom binary outputs
+# on the CI. $IREE_BINARY_DIR may not be setup right or the object files may
+# not be getting deployed to the test_all/test_gpu bots.
+excluded_tests+=(
+  "iree/samples/custom_dispatch/cpu/embedded/example_hal.mlir.test"
+  "iree/samples/custom_dispatch/cpu/embedded/example_stream.mlir.test"
+)
```

Comment:
Rather than exclude in this script (which CI runs but users are not required to), how about editing the source: https://github.com/iree-org/iree/blob/1b72894442d95944da535ea09271aadc6adf68d3/samples/custom_dispatch/cpu/embedded/CMakeLists.txt#L67-L80

Options:
* Comment out the test 
* Set to `DISABLED` or `WILL_FAIL`
  * https://cmake.org/cmake/help/latest/command/set_tests_properties.html
  * https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#properties-on-tests

Reply:
Oooh.. Weird. Okay, this exclusion style is alright then...